platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -49,6 +51,9 @@ export default class HangingProtocolService extends PubSubService {
   studies: StudyMetadata[];
   // stores all the protocols (object or function that returns an object) in a map
   protocols: Map<string, Protocol>;
+  // protocols copy for the case where protocols has computed values which override
+  // the original values
+  protocolsCopy: Map<string, Protocol>;
```

Comment:
Could we call one of them protocolsCustomized and the other just protocols?  protocolsCopy is really unclear to me as to what it means.